index.html

```diff
@@ -0,0 +1,203 @@
+                    <p>
+                        Connect with people across the globe and learn from them.
+                    </p>
+                </div>
+
+                <div class="bg-white/10 p-4 flex md:flex-col justify-center items-center text-center">
+                    <i class="hover:text-[#f28f10] fa-solid fa-earth-americas fa-2x md:fa-3x"></i>
+                    <h4 class="text-[#f28f10] hover:text-white text-xl px-2 py-2">COURSE</h4>
+                    <p>
+                        Connect with people across the globe and learn from them.
+                    </p>
+                </div>
+
+            </div>
+
+            
+                <a class="hidden md:block" href="#">
+                    <button type="button" class="my-14 hover:text-[#f28f10] underline text-lg hover:underline">
+                        SEE THE WHOLE PROGRAM
+                    </button>
+                </a>
+
+                <a class="block md:hidden" href="#">
+                    <button type="button" class="my-14 text-lg text-white bg-[#f28f10] p-4 active:border-white active:border-2 ">
+                        JOIN PROGRAM
+                    </button>
+                </a>
+            
+
+        </section>
+
+<!-- SPEAKERS -->
+
+<section class="flex justify-center items-center flex-col py-20 px-[4vw]" >
+    <h2 class="mp-txt text-2xl md:text-4xl font-semibold hover:text-[#f28f10]">
+        SPEAKERS
+    </h2>
+    <div class="mp-underline"></div>
+    <div class="grid grid-cols-1 md:grid-cols-2 justify-center mt-14" id="speaker-section">
+        <!-- SPEAKERS INJECTED -->
+    </div>
+</section>
+
+<!-- PARTNERS -->
+
+<section class="bg-[#272a31] text-white flex justify-center items-center flex-col py-10">
+    <h2 class="mp-txt text-2xl md:text-4xl font-semibold hover:text-[#f28f10]">
+        PARTNERS
+    </h2>
+
+    <div class="mp-underline"></div>
+
+    <div class="mt-14 w-full flex justify-center items-center gap-14 flex-col md:flex-row">
+
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/bmw-logo-svgrepo-com.svg" alt="bmw logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/apple-logo-svgrepo-com.svg" alt="apple logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/google-icon-logo-svgrepo-com.svg" alt="google logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/instagram-2-1-logo-svgrepo-com.svg" alt="insta logo">
+    <img class="h-14 hover:animate-spin" src="./Assets/partners/whatsapp-icon-logo-svgrepo-com.svg" alt="wa logo">
+
+    </div>
+
+</section>
+
+    </main>
+
+<!-- FOOTER SECTION -->
+
+<footer class="flex w-full justify-center p-2 gap-2 items-center my-8">
+    <div class="flex justify-end w-1/2">
+    <img class="h-20 object-contain" src="./Assets/partners/CreativeCommons_logo_trademark.svg" alt="cc logo">
+    </div>
+    <div class="w-1/2">
+    <h2 class="hover:text-[#f28f10]">© All RIGHTS RESERVED 2022</h2>
+    </div>
+</footer>
```

Comment:
Kindly make sure that your project sticks as much as possible to the template provided. The desktop footer section in your project is different from the template provided. You could use Lorem ipsum text to generate the missing text.
Your design
![image](https://user-images.githubusercontent.com/98363075/203321572-937b63d9-33b4-4636-bc30-6431b14544cf.png)

Template design
![image](https://user-images.githubusercontent.com/98363075/203321663-aed94d04-ba44-4dfa-8851-0f043cf0d57d.png)